bbc/simorgh
# WSTEAM1-888 - Removes --all flag from production install modules script

Resolves JIRA [WSTEAM1-888](https://jira.dev.bbc.co.uk/browse/WSTEAM1-888)

Overall changes
======
This removes the `--all` flag from the script used in our release pipeline, which would install production dependencies from ALL workspaces including our Next.js app. This prevents our Next.js app dependencies from leaking into the deployment of our Express server.

Code changes
======

- Removes `--all` flag from node modules install script

Testing
======
1. Run the bash script `scripts/installNodeModules.sh --production`
2. Check that next.js dependencies don't leak into the root node_modules folder

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
